kubernetes/cray-bos/values.yaml.in

```diff
@@ -8,9 +8,12 @@
 #   pullPolicy: "" (default = "IfNotPresent")
 
 boa_image:
-  repository: cray/cray-boa
-  tag: IMAGE_CRAY_BOA_TAG
-
+  repository: artifactory.algol60.net/csm-docker/stable/cray-boa
```

Comment:
Do we want boa_image to have an 'image' attribute to be symmetric with 'database'?

Reply:
Given that BOA is going away, I don't particularly care much.